java/src/jmri/jmrit/display/Editor.java

```diff
@@ -464,6 +464,16 @@ protected final void setPaintScale(double newScale) {
     ToolTipTimer _tooltipTimer;
 
     protected void setToolTip(ToolTip tt) {
+        if (tt != null) {
+            var pos = tt.getPositionable();
+            if (pos != null) {  // LE turnout tooltips on not have a Positionable
```

Comment:
`// LE turnout tooltips on not have a Positionable`

This sentence looks a bit strange to me. Should `on` be `do` instead?